extensions/notification/queueProcessor/QueueProcessor.js

```diff
@@ -124,6 +140,9 @@ class QueueProcessor extends EventEmitter {
                         'notification entries');
                     this.emit('ready');
                 });
+                // callbackify getConfig from notification config manager
```

Comment:
what is the requirement for callbackifying? We should be moving away from callbacks to promises.